syncto/views/errors.py

```diff
@@ -18,11 +19,8 @@ def response_error(context, request):
                              context.response.reason,
                              context.response.text)
 
-    statsd = request.registry.statsd
-
-    if statsd:
-        statsd.count("syncclient.status_code.%s" %
-                     context.response.status_code)
+    statsd_count(request, "syncclient.status_code.%s" %
+                 context.response.status_code)
```

Comment:
The idea was to log every status code (wrong or right) just have some metrics. (ie if we see that we have 70% of 304, maybe we can decide to lower the synchronization frequency or decide to prioritize the push notification before sync)